content/docs/rfcs/43/README.md

```diff
@@ -78,7 +78,7 @@ d.   -> sA, sAeB, sAsB  {s}
 
 ### Protocol Flow
 
-1. The device `B` exposes through a QR code a Base64 serialization of:
+1. The device `B` exposes through a QR code a [base64 (url safe)](https://datatracker.ietf.org/doc/html/rfc4648#section-5) serialization of:
```

Comment:
```suggestion
1. The device `B` exposes through a QR code a [base64 (with padding)](https://datatracker.ietf.org/doc/html/rfc4648) serialization of:
```

Reply:
In this particular case, base64 url safe is needed! the QR code is used as part of the query string of an URL, so it's possible that a generated QR code ends up containing a character that ends up breaking noise implementation of js-waku

Reply:
Aw! I see 🙂 👍🏼

Reply:
Check this out: https://examples.waku.org/noise-js

Here's an example URL generated in here: https://examples.waku.org/noise-js/?messageNameTag=60dbfc59be265d1a44c101cac6ba4a18&qrCode=d2FrdS1ub2lzZS1zZXNzaW9ucw%3D%3D%3AMC4x%3AMTA%3D%3AaKz7rvb%2BnjJSE1fsmvi0ZiJ03bfXxDv7zi5AmuAvyXQ%3D%3AtVibhlMutOcWoyK9oFVO0XL0F897h8aCyvePtGpXWk8%3D